README.md

```diff
@@ -26,35 +28,62 @@ This project provides an easy to use developers environment for the **RIF Market
 
 ## Part of tutorial
-These will be installed during the tutorial
+These repositories will be cloned and installed during the tutorial
 
-1. [RIF Marketplace Developer Environment](https://github.com/rsksmart/rif-marketplace-dev/) project
-1. [RIF Comms Pubsub BootNode](https://github.com/rsksmart/rif-communications-pubsub-bootnode) project   
+1. [RIF Marketplace Developer Environment](https://github.com/rsksmart/rif-marketplace-dev/) project  
+1. **(Disabled)** [RIF Comms Pubsub BootNode](https://github.com/rsksmart/rif-communications-pubsub-bootnode) project  
 1. [RIF Marketplace Cache](https://github.com/rsksmart/rif-marketplace-cache/) project
 1. [RIF Marketplace Upload Service](https://github.com/rsksmart/rif-marketplace-upload-service/) project
+1. [RIF Storage Pinning Service](https://github.com/rsksmart/rif-storage-pinner/) project
 1. [RIF Marketplace UI](https://github.com/rsksmart/rif-marketplace-ui/) project
 1. [RNS Manager Project](https://github.com/rnsdomains/rns-manager-react)
-1. [RIF Storage Pinning Service](https://github.com/rsksmart/rif-storage-pinner/) project
+1. [RIF Notifier Service](https://github.com/rsksmart/rif-notifier) project
+
+# Quick Start
+
+1. `meta git clone git@github.com:rsksmart/rif-marketplace-dev.git` - will clone the meta dev repository and recursively clone the nested ones. In case you cloned the repo with the standard git command, simply execute - `meta git update` inside of the repo.
+2. Execute `npm run all` or `npm run mkp`
+- `npm run all` - will install all the dependencies, configure project with default options for the local setup, and start services and applications via `pm2` manager.
+- `npm run mkp` - will do the same, but only for the contracts, cache and dapp.
+3. Use `pm2` to see the list of running processes (`pm2 list`), logs(`pm2 logs <id|name>`), etc.
+You should see the result similar to this one
+![pm2-list](/assets/images/pm2-list.png)
+
 
-# Setup:
+`meta` tool provides a convenient interface to work with multiple repositories simultaneously. There are different plugins for bulk operations, including `git` and `npm`. Some examples are:
+- `meta git status`, `meta git status --include-only dir1,dir2`
+- `meta git update`
+- `meta git -b checkout branch-name`
+- `meta npm install`
+- `meta exec "<any-command>"`
+
+Please refer to the documentation for more details.
+
+
+# Detailed Setup:
 ## 1. Developers Environment
 Download and setup the RIF Marketplace Developer Environment
 ```
-git clone git@github.com:rsksmart/rif-marketplace-dev.git
+meta git clone git@github.com:rsksmart/rif-marketplace-dev.git
 
 cd rif-marketplace-dev
 ```
 
-### 1.1. Starting docker
-Now you can start docker with
+### 1.1. Starting local blockchain
+Now you can start local blockchain in terminal, pm2 or docker with
+```
+npm run ganache-cli or npm run ganache-cli:mining or npm run ganache-cli:pm2
+```
+
```

Comment:
might be easier to read if it is split:

Now you can start local blockchain in
- terminal:
```
npm run ganache-cli
```
- pm2:
```
npm run ganache-cli:pm2
```
- or docker:
```
npm run ganache-cli:mining
```


btw, why is it "mining" for docker?

Reply:
nope, it's a terminal as well, the docker is  below